dist/emoji-en-US.json

```diff
@@ -12443,5 +12443,41 @@
   "🏳️‍⚧️": [
     "transgender flag",
     "lgbtq"
+  ],
+  "😶‍🌫️": [
+    "face in clouds",
+    "shower",
+    "steam",
+    "dream"
+  ],
+  "😮‍💨": [
+    "face exhaling",
+    "relieve",
```

Comment:
Would it make sense to use `relief` here, given use of `relief` in the `grinning_face_with_sweat` emoji?

https://github.com/muan/emojilib/blob/main/dist/emoji-en-US.json#L63